.github/workflows/stale.yml

```diff
@@ -0,0 +1,17 @@
+name: Mark stale pull requests
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/stale@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-pr-message: "A reminder that this pull request has had no activity for 30 days"
+          stale-pr-label: "no-pr-activity"
+          days-before-stale: 30
```

Comment:
```suggestion
          days-before-stale: 14
```
:-)?

Reply:
You're keen ;)

I really don't mind, but that seems pretty quick? 21? Meet in the middle?

Reply:
I don't really mind either, so as you like - I just thought 30 days is actually a lot of inactivity.